apps/mobile/src/features/Settings/useCases/changeAppSettings/About/About.tsx

```diff
@@ -22,31 +20,87 @@ export const About = () => {
   })
 
   return (
-    <View style={styles.about}>
-      <View style={styles.row}>
-        <Text style={styles.labelText}>{strings.currentVersion}</Text>
+    <View
+      style={{
+        ...a.flex_1,
+        backgroundColor: p.bg_color_max,
+        ...a.p_lg,
+      }}
+    >
+      <View
+        style={{
+          ...a.flex_row,
+          ...a.justify_between,
+          ...a.py_lg,
+        }}
+      >
+        <Text
+          style={{
+            color: p.gray_900,
+            ...a.body_1_lg_medium,
+          }}
```

Comment:
I think it should be a `ta` need to check with design
```suggestion
          style={[
            {color: p.gray_900},
            a.body_1_lg_medium,
          ]}
```

Reply:
body_1_lg_medium is attached to "a"